src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Diagnostics/CompositionTraceSource.cs

```diff
@@ -0,0 +1,58 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// -----------------------------------------------------------------------
+using System;
+using Microsoft.Internal;
+
+namespace System.ComponentModel.Composition.Diagnostics
+{
+    internal static class CompositionTraceSource
+    {
+#if FEATURE_TRACING
+        private static readonly TraceSourceTraceWriter Source = new TraceSourceTraceWriter();
```

Comment:
@weshaggard on .NET Core we have TraceSource so I assume we should flip this define so that TraceSourceTraceWriter is the default?

Or just delete DebuggerTraceWriter ?

Reply:
Yes we can flip to just TraceSource.